### PR TITLE
Added provision to remove specific files for a given dataset from SNTSample

### PR DESCRIPTION
**Pull request by bsathian**
Very primitive stuff

### CODE REVIEW

**Comment by bsathian** [comment]
Doing whatever I can for the tools I use :)

**Comment by bsathian** [comment]
Removed the hardcoded stuff now

File: metis/Sample.py [hunk 1]
```diff
@@ -340,16 +340,9 @@ def get_files(self):
...
             for filename in self.skip_files:
-                if "merged_ntuple" not in filename:
-                    filename = "merged_ntuple_"+filename
-                if ".root" not in filename:
-                    filename = filename+".root"
-                full_file_path = os.path.join(self.get_location(),filename)
-                if full_file_path in filepaths:
-                    print("Removing {} from list".format(full_file_path))
-                    filepaths.remove(full_file_path)
+                    print("Removing {} from list".format(filename))
```
Comment (aminnj) [inline metis/Sample.py]: Can you use 
`self.logger.info()` instead of `print()` here

File: metis/Sample.py [hunk 2]
```diff
@@ -340,16 +340,9 @@ def get_files(self):
...
+                    filepaths.remove(filename)
                 else:
                     print("{} not in list of files. Skipping its deletion".format(filename))
```
Comment (aminnj) [inline metis/Sample.py]: and same as above

Comment (bsathian) [reply metis/Sample.py]: Changed

**Comment by aminnj** [comment]
Thanks!

**Pull request merged**